Make idle-issues workflow customizable (#34)
mdn/workflows: README.md

- This `input` is optional with a default of an empty string

### stale-days

The number of days before the issue or pull request is considered idle and the label and/or comment is applied.

- This `input` is optional with a default of 37

### closure-days

The number of days before the idle issue or pull request is closed.  Set to -1 to disable.

- This `input` is optional with a default of -1

## Secrets

This action requires the following secrets to be passed by the caller. These need to be set as [environmental secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) using the calling repositories settings.

### GH_TOKEN

Personal access token passed from the caller workflow. Read the documentation on [creating a PA token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token).

## Usage

> In the repository that will call this action, you will need to add a `.github/workflows/pr-rebase-needed.yml` file with the following content:

Reviewer: ```suggestion
In the repository that will call this action, you will need to add a `.github/workflows/mark-as-idle.yml` file with the following content:
```

### With defaults

```yml
name: "PR Needs Rebase"

on:
  push:
  pull_request_target:
    types: [synchronize]

jobs:

>   pr-needs-rebase:
>     uses: mdn/workflows/.github/workflows/pr-needs-rebase.yml@main

Reviewer: ```suggestion
  mark-as-idle:
    uses: mdn/workflows/.github/workflows/mark-as-idle.yml@main
```

    secrets:
      GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

### Overriding some defaults

```yml
name: "PR Needs Rebase"

on:
  push:
  pull_request_target:
    types: [synchronize]

jobs:

>   pr-needs-rebase:
>     uses: mdn/workflows/.github/workflows/pr-needs-rebase.yml@main

Reviewer: ```suggestion
  mark-as-idle:
    uses: mdn/workflows/.github/workflows/mark-as-idle.yml@main
```
